actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/actionable_error"
+require "rails/command"
+
+module ActionMailbox
+  # Generic base class for all Action Mailbox exceptions.
+  class Error < StandardError; end
+
+  # Raised when we detect that Action Mailbox has not been initialized.
+  class SetupError < Error
+    include ActiveSupport::ActionableError
+
+    def initialize(message = nil)
+      return super if message
```

Comment:
I find this early return difficult to understand. Why would there already be a message here?

Would it be clearer with?

```ruby
message ||= <<~MESSAGE
…
MESSAGE

super
```

Reply:
Of course!